### DISCUSSION TITLE
Every time I click to my profife page link without being logged in, they write 404: this is not the page you are looking for.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110814] [Other Feature Feedback, Questions, & Ideas]
I'm new to github.

It has been 3 weeks since I created my account, but every time I click to my profife page link without being logged in, they write 404: this is not the page you are looking for.

With this problem I will not be able to share my profile with people.

This is the link https://github.com/ernest-mm

Please help me.

**UPDATE: I’ve already solved this issue. It turns out my account was flagged**.
So I created a GitHub ticket explaining my problem and they cleared the restrictions.

**Comment by tanisha0605** [comment]
> I'm new to github.
> 
> It has been 3 weeks since I created my account, but every time I click to my profife page link without being logged in, they write 404: this is not the page you are looking for.
> 
> With this problem I will not be able to share my profile with people.
> 
> This is the link https://github.com/ernest-mm
> 
> Please help me.
> 
> **UPDATE: I’ve already solved this issue. It turns out my account was flagged**. So I created a GitHub ticket explaining my problem and they cleared the restrictions.



How long did it take for them to respond to the ticket, this same thing happened to me and my ticket is not responded to yet since a week

**Reply by ghost** [reply]
It was less than a week

**Reply by Nora-rad** [reply]
I have the same problem. but they don't answer my tickets. my ticket hasn't been answered for a month.
